automated-comments/java/general/constructor_too_long.md

```diff
@@ -0,0 +1,6 @@
+At least one constructor in this solution is getting long.
```

Comment:
Is it possible to identify the constructor in question and name it? Then this can be written less vague.

Reply:
And perhaps also explain what is meant by "long" (too many arguments, arguments with large names, too much lines of code)?